packages/server/lib/controllers/sync.controller.ts

```diff
@@ -373,6 +374,11 @@ class SyncController {
             }
 
             const provider = await configService.getProviderConfig(providerConfigKey, environmentId);
+            if (!provider) {
+                res.status(400).json({ error: { code: 'not_found' } });
+                return;
+            }
+            const syncConfig = await getSyncConfigRaw({ environmentId, config_id: provider.id!, name: action_name, isAction: true });
```

Comment:
What if this isn't found or comes back as null?

Reply:
indeed, forgot about this one 👌🏻